website/docs/components/form/index.md

```diff
@@ -27,7 +27,7 @@ Ten eerste wordt een invoerveld gevalideerd als de focus wordt verplaatst. Er ve
 ### Inline validatie
 
 **Verplichte velden**
-Op het moment dat de gebruiker focus verplaatst uit het verplichte invoerveld of het formulier indient, verschijnt er een foutmelding direct onder het betreffende invoerveld: dit is een `.dso-message` waar met `@aria-describedby` naar verwezen moet worden. Deze vorm van validatie wordt toegepast op alle formulierelementen.
+Op het moment dat de gebruiker focus verplaatst uit het verplichte invoerveld of het formulier indient, verschijnt er een foutmelding direct onder het betreffende invoerveld: dit is een `.dso-message` waar met `@aria-describedby` naar verwezen moet worden. De `.dso-message` krijgt een `role="alert"`-attribuut. Deze vorm van validatie wordt toegepast op alle formulierelementen.
```

Comment:
Moeten we rekening houden met meerdere `role="alert"`s op een pagina?

Bv: Een gebruiker vult 'n veld verkeerd in, negeert de foutmelding en vult nog een veld verkeerd in?

Of de klassieker: Een formulier met verplichte velden. Het eerste veld wordt foutief ingevuld, de gebruiker drukt op TAB, krijgt een foutmelding op veld 1 en heeft focus in veld 2. De gebruiker focust veld 1 (en dus blurred veld 2), veldvalidatie gaat af op veld 2. De gebruiker heeft nu twee foutmeldingen.

Reply:
Het zal nooit ideaal worden. deze meldingen kunnen door de gebruiker te allen tijde gestopt worden. Diverse scenario's gistermiddag met @timveld doorgesproken. Dit is de beste oplossing, met name als we NVDA (ons referentieplatform) willen meenemen.

Reply:
Ik bedoelde eigenlijk of we moeten voorschrijven dat er maar 1 `role="alert"` actief is, en dat de afnemer de `role="alert"` moet weghalen als er een andere foutmelding ontstaat?

Reply:
Er wordt er maar 1 tegelijk de dom in geschoten als het goed is. het ligt aan de interactie hoe chaotisch het kan worden, maar dat is aan de gebruiker imho. Een gebruiker die gestructureerd de velden afloopt, en een paar foutjes maakt, moet denk ik een soepele ervaring hebben zo.

Reply:
> Er wordt er maar 1 tegelijk de dom in geschoten als het goed is.

Ja, akkoord.

Ik vraag mij af of het kwaad kan als er meerdere tegelijkertijd in het DOM zijn.

Reply:
Nope, zolang ze niet 'exposed' zijn gaat het goed.

Reply:
Alright! Dank